app/javascript/components/project-submissions/components/submission.jsx

```diff
@@ -62,23 +62,25 @@ const Submission = forwardRef(({
         {isCurrentUsersSubmission
           ? (
             <div
```

Comment:
Instead of breaking this out into its own component, I can also just defined the classes inside the file and remove them from Tailwind component styles. Thoughts?

Reply:
Sounds good to me @leosoaivan